scripts/generate_grpc

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -e
+
+cd $(dirname $0)/..
+
+# check and download dependency for gRPC code generate
+if [ ! -e ./protobuf/vendor/protobuf/src/google/protobuf ]; then
+    rm -rf ./protobuf/vendor/protobuf/src/google/protobuf
+    DIR="./protobuf/vendor/protobuf/src/google/protobuf"
+    mkdir -p $DIR
+    wget https://raw.githubusercontent.com/protocolbuffers/protobuf/v24.3/src/google/protobuf/empty.proto -P $DIR
```

Comment:
Make `v24.3` as a global variable `PROTOBUF_VER`

Reply:
Updated it. Thank you.